src/index.ts

```diff
@@ -2,4 +2,5 @@
 export * as keyStores from './key_stores/index';
 export * from './common-index';
 export * from './connect';
-export * from './constants';
\ No newline at end of file
+export * from './constants';
+export {NEAR, Gas} from 'near-units';
```

Comment:
Should we maybe also export `parse` from `near-units`? Maybe this:

```suggestion
export {NEAR, Gas, parse as parseUnits} from 'near-units';
```